docs/contributing.md

```diff
@@ -4,7 +4,7 @@ First you need to get familiar with the [Architecture guide](architecture.md), w
 from a high perspective how everything works together.
 
 After our experience building the [Enterprise MongoDB Kubernetes
-Operator](https://github.com/mongodb/mongodb-enterprise-operator), we have
+Operator](https://github.com/mongodb/mongodb-enterprise-kubernetes), we have
```

Comment:
this link is actually correct! We have multiple kubernetes operators. (Enterprise + Community)

Reply:
uhm are you sure, the correct enterprise link is https://github.com/mongodb/mongodb-enterprise-kubernetes

Reply:
ignore me, this change is correct! Thanks @killian2k

Reply:
> Thanks for this fix!
> 
> But it looks like your commits are not signed, can you make sure they are?
> 
> https://docs.github.com/en/github/authenticating-to-github/managing-commit-signature-verification/signing-commits
> 
> we also have a page on our private wiki, you can check it out

Thanks for the link, it should be fine now.